content/docs/getting-started/installing.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Installing
+weight: 2
+---
+
+## From extension stores
+
+Scratch Addons is available in both Chrome Web Store and Add-ons for Firefox.
```

Comment:
```suggestion
Scratch Addons is available in both Chrome Web Store and addons.mozilla.org.
```
"addons.mozilla.org" is the official name of the extension distribution platform for Firefox.